docs/Getting Started Part 2/index.md

```diff
@@ -0,0 +1,323 @@
+  * `UI` - Interface responsible for RPC events and information made visible to the user. Includes `Show` and `Alert`. 
+  * `Buttons` - Interface responsible for RPC events and information related to hard and soft buttons in the vehicle. Includes `OnButtonPress` and `OnButtonEvent`.
+  * `VR` - Interface responsible for RPC events and information related to voice recognition. Contains RPC voice recognition counterparts to the UI component RPCs.
+  * `TTS` - Interface responsible for RPC events and information related to text to speech capabilities. Includes RPC `Speak`.
+  * `Navigation`- Interface responsible for RPC events and information related to navigation. Includes `StartStream` and `GetWayPoints`.
+  * `VehicleInfo`- Interface responsible for RPC events and information related to vehicle data. Includes `GetVehicleData` and `SubscribeVehicleData`.
+  * `RC` - Interface responsible for RPC events and information related to the [Remote Control Feature](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0065-remote-control.md).
+  * `AppService` - Interface responsible for RPC events and information related to the [App Services Feature](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0065-remote-control.md).
+
+### Response
+
+SDL Provides a JSON Response
+
+| Key | Value Info   |
+| :------------- | :------------- |
+| "id"      | The value from the corresponding request      |
+| "result" | Value of id multiplied by 10. HMI can treat this as a successful registration|
+
+Example Response:
+``` json
+{
+  "id": 100,
+  "jsonrpc": "2.0",
+  "result": 1000
+}
+```
+
+## Component Readiness Requests 
+
+Once the components are registered, the HMI must notify SDL Core that it is ready to begin further communication using the [BasicCommunication.OnReady](../basiccommunication/onready) notification.
+
+Upon receipt of the OnReady notification, SDL Core will begin checking the availability of the different HMI components via a chain of requests:
+
+  * `UI.IsReady` - The display availability
+  * `VR.IsReady` - The voice recognition module availability
+  * `TTS.IsReady` - The Text-To-Speech module availability
+  * `Navigation.IsReady` - Navigation engine availability
+  * `VehicleInfo.IsReady` - Indicates whether vehicle information can be collected and provided
+  * `RC.IsReady` - Indicates whether vehicle RC modules are present and ready to communicate with SDL Core
+
+!!! note
+
+In the case of a WebSocket connection, RPCs to each of the components are sent within a separate WebSocket connection.
+
+!!!
+
+|||
+IsReady Sequence
+![IsReady Sequence](./assets/IsReadySequence.png)
+|||
+
+!!! info
+
+If the response to any of the component `IsReady` requests contains `{"available": false}`, SDL Core will no longer communicate with that component.
+
+!!!
+
+ 
+## Registering for Notifications
+The HMI must also register for notifications individually using the following RPC format
```

Comment:
```suggestion
The HMI must also register for notifications individually using the following RPC format.
```
i think there should be some punctuation here, a period is used in the Response section